fhir-persistence-schema/README.md

```diff
@@ -141,6 +141,23 @@ Edit `wlp/usr/servers/fhir-server/config/default/fhir-server-config.json` and ad
 --tenant-key "<the-base64-tenant-key>"
 ```
 
+### Add a Key to Existing Tenant
+To add a tenant key for an existing tenant, replace FHIRDATA with your client schema, and TENANT_NAME with your tenant's name. 
+
+```
+--prop-file db2.properties
+--schema-name FHIRDATA
+--add-tenant-key TENANT_NAME
```

Comment:
The other snippets in this README use `default` as the tenant name.
I think we should either do the same here (or update everywhere to TENANT_NAME).